packages/shopify-codemod/test/transforms/simplify-?=-if-else-statement.test.js

```diff
@@ -0,0 +1,8 @@
+import 'test-helper';
```

Comment:
`conditional-assign-to-if-statement` references `a || a = b` as a `Conditional Assignment`. How would we differentiate between that and `a ?= b`?